packages/flutter_goldens/test/flutter_goldens_test.dart

```diff
@@ -511,50 +505,6 @@ void main() {
           );
         });
       });
-
```

Comment:
We don't need it anymore because we are no longer retrieving digests. :)
Before this change, we had to look up all of the digests and then figure out which one we needed in order to get the right image. With the new endpoint, we can just ask for the exact image we need.